server/src/services/initialization/onInitialize.ts

```diff
@@ -6,7 +6,6 @@ import {
 } from "vscode-languageserver/node";
 import { ServerState } from "../../types";
 import { tokensTypes } from "../semanticHighlight/tokenTypes";
-import { isSlangSupported } from "../../parser/slangHelpers";
```

Comment:
Is there no equivalent in Slang now?

Reply:
I removed it because as I understand, the wasm build works on every os and arch